docs/mutation-data-transcript-annotation.md

```diff
@@ -0,0 +1,85 @@
+## Mutation Data Transcript Annotation 
+
+This document describes how each mutation in cBioPortal gets annotated with a specific gene symbol + protein change. 
+
+### Biological Background 
+
+This section explains the concepts of protein isoforms and transcripts. 
+
+#### What is an isoform? 
+
+From a single gene (string of nucleotides) multiple protein sequences can be formed (string of amino acids). For 
+example: parts of the gene that code for proteins (exons) can be included or excluded. Each of these different proteins 
+is called an [isoform](https://en.wikipedia.org/wiki/Protein_isoform). A single mutation can change the amino acids in 
+each of these these isoforms differently. E.g. in one isoform it might change a P to a T, but in the other isoform that 
+particular exon does not get included and it is therefore not changing the amino acid sequence at all. In cBioPortal 
+for convenience sake we assign a single gene symbol + protein change to each mutation. For most cases this works well 
+because there is only one protein isoform relevant in a clinical setting. There are of course exceptions and we are 
+therefore working on improving this representation. An explanation of the relation between transcripts and protein 
+isoforms can be found in the next section. 
+
+#### What is a transcript? 
+
+DNA is transcribed to an mRNA transcript which is then translated to a protein sequence. A transcript is thus 
```

Comment:
This is a little unclear to me.  I think what you are saying is that the variability that results in multiple isoforms occurs during transcription process. I.e. multiple transcripts are possible from the same gene and those map 1 to 1 with isoforms?

Reply:
exactly - im not sure tho about the 1 to 1 cause I could imagine theoretically it's possible multiple transcripts produce the same isoform